README.md

```diff
@@ -0,0 +1,103 @@
+## Introduction
+This is a demo module for Islandora CLAW. It illustrates the modeling of a content type and creation of a drupal module for CLAW.
+
+
+## Steps to create claw_porcus
+### 1 - Create a new Media bundle
+* Go to `/admin/structure/media/add`
+* Provide a label to the bundle: PORCUS_TEXT
+* Leave the type provider as Generic media
+* Go to the Manage fields of the new media bundle: `/admin/structure/media/manage/porcus_text/fields`
+* Add a new file field
+
+### 2 - Create a new Fedora Resource type bundle
+* Go to `/admin/structure/fedora_resource_type`
+* Provide a label to the fedora resource type: Islandora Porcus Object
+* Go to Manage fields of the new fedora resource type: `/admin/structure/fedora_resource_type/islandora_porcus_object/edit/fields`
+* Add a new description field (Text (plain, long))
+* Add a new field Reference type Other
+* Chose 'Type of item to reference' as Media
+* Select the Default 'Reference method' and PORCUS_TEXT as the 'Bundles' 
+* Go to Manage form display: `/admin/structure/fedora_resource_type/islandora_porcus_object/edit/form-display`
+* Specify the porcus_text WIDGET as Inline entity form - complex
```

Comment:
We should sat to use the `inline` form mode in the dropdown you get from clicking on the gear at the end of the row.